config/app.php

```diff
@@ -244,14 +243,12 @@
     | this array to grant expanded functionality to your applications.
     |
     */
-
-    'providers' => array_merge(include(base_path('modules/system/providers.php')), [
-
+    
+    'providers' => array_merge(include base_path('modules/system/providers.php'), [
         // 'Illuminate\Html\HtmlServiceProvider', // Example
-
-        'System\ServiceProvider',
+        'System\\ServiceProvider',
```

Comment:
I'm not sure why this was changed, is it a fluke that the original value worked? Or is the printer misunderstanding that backslashes are only required to be escaped in single quote strings if they're right next to a value that would otherwise have to be escaped (i.e. a single quote)?

Reply:
When you pass a variable it'll pass the value of that variable, i.e. if you were to pass `CURLOPT_CRLF` it would set the value 27.

To set a namespace you'd either need to use `\\\` as a separator or you could use the `constant()` method and set the full class reference, i.e. `Path\Name\MyClass::class`